src/js/components/container/SettingsButton.jsx

```diff
@@ -24,20 +27,28 @@ class SettingsButton extends Component {
     })
   }
 
-  renderSetting (setting) {
-    // Currently all settings are boolean. In order to support generic settings, this option needs to be expanded
+  renderSetting(setting) {
+    var settingControl = null;
+    switch(typeof(setting.value)) {
+      case 'boolean':
+        settingControl = <Switch setting={setting} onUpdate={() => Settings.set(setting.key, !setting.value)}></Switch>
+        break;
+      
+      case 'number':
+        settingControl = <NumericSlider setting={setting} 
+        onUpdate={(e) => Settings.set(setting.key, parseInt(e.target.value))}
+        min="1" max="5"></NumericSlider>

```

Comment:
You are setting this here to be 1 to 5
But where are you setting thia for scroll speed?

Reply:
I'm not sure I understand the question. The 1 and 5 are the limits for the slider. 
when the slider is changed, the onUpdate callback is called, changing the setting. 
In the InfinateTable the scroll speed is changed by adjusting how much is added to the tables margin-top using the newScroll method.

Reply:
So the scroll value wil be between 1 and 5

Reply:
Yes, that's good.
@esh4 please merge this. If you can't for some technical reason, tell me and I'll merge it.

Reply:
The branch is failing AppVeyor checks, so it's not letting me merge

Reply:
Ok. I'll merge it